tests/test-cases/Group6-VIC-Machine/6-03-Delete.md

```diff
@@ -73,3 +73,23 @@ This test requires that a vSphere server is running and available
 
 ### Expected Outcome:
 1. All steps should succeed
+
```

Comment:
Extra space?

Reply:
Followed the formatting style of this file - there are two lines between tests here.